_data/sites.json

```
@@ -9647,6 +9647,16 @@
         ]
     },
 
+    {
+        "name": "Spreadshirt",
+        "url": "https://my.spreadshirt.com/account/contact",
+        "difficulty": "medium",
```

```suggestion
        "difficulty": "easy",
```
I think this one can be easy, you can do it all on the same page with all but a few clicks and types, looks fine.